addons/web/static/src/views/view_compiler.js

```diff
@@ -243,13 +244,29 @@ export class ViewCompiler {
         return compiled;
     }
 
+    applyTechnical_chm(root) {
+        const stack = [root];
+        for (const node of stack) {
```

Comment:
you should rather do a t-if here instead of relying on the invisible modifier. The condition can simply read inside fieldInfo (there you'll have the technical_chm attribute if it has been set in the arch). That way, no need to add technical_chm_something to the eval context.